sys/geom/concat/g_concat.c

```diff
@@ -1107,8 +1107,6 @@ g_concat_ctl_append(struct gctl_req *req, struct g_class *mp)
 		gctl_error(req, "No 'arg%u' argument.", 1);
 		goto fail;
 	}
-	if (strncmp(name, "/dev/", strlen("/dev/")) == 0)
-		name += strlen("/dev/");
```

Comment:
This loses the effect of `name` being changed.  Notice `name` is potentially used in several more places and this loses that effect.

Reply:
This change is good. We use name twice. g_provider_by_name does the right thing. As does g_concat_find_disk. The latter should be recoded in the same style as the former, but this code can be removed here. The error message changing as a result are of no consequence.